test/torchaudio_unittest/common_utils/case_utils.py

```diff
@@ -98,3 +98,5 @@ def skipIfNoModule(module, display_name=None):
 skipIfNoCuda = unittest.skipIf(not torch.cuda.is_available(), reason='CUDA not available')
 skipIfNoSox = unittest.skipIf(not is_sox_available(), reason='Sox not available')
 skipIfNoKaldi = unittest.skipIf(not is_kaldi_available(), reason='Kaldi not available')
+skipCUDAIfRocm = unittest.skipIf(os.getenv('PYTORCH_AUDIO_TEST_WITH_ROCM', '0') == '1',
```

Comment:
1. Instead of `PYTORCH_AUDIO`, `TORCHAUDIO` is the keywords we use everywhere.
2. This logic looks like it's inverted. When the `PYTORCH_AUDIO_TEST_WITH_ROCM` is `1`, it evaluates to `unittet.skipIf(True, ...)`. Which sounds counter intuitive. Am I reading it wrong?

Reply:
1. I will make the change.
2. That is the functionality that we want. We do a similar thing in pytorch. We skip some tests on rocm even though they work on cuda. I can rename it to skipIfRocm to be more clear.

Reply:
Oh I see. I was indeed reading it wrong. Thanks for the clarification.